ci/docs.yml

```diff
@@ -18,7 +18,7 @@ dependencies:
     - ipython
     - jupyter_client
     - matplotlib-base
-    - sphinx-book-theme
+    - sphinx-book-theme<=0.13.1 # pydata-sphinx-theme 0.13.2 broke sphinx_book_theme import
```

Comment:
I don't believe this is necessary anymore.

Reply:
Looks so!